CMakeLists_files.cmake

```diff
@@ -335,6 +336,7 @@ if(ENABLE_ECL_INPUT)
     tests/SPE1_TESTCASE.F0025
     tests/SPE1_TESTCASE.X0025
     tests/SPE1CASE1.UNSMRY
+    tests/SPE1CASE1A.UNSMRY
```

Comment:
Is it necessary to add a new binary summary case - could you not use the already existing `SPECASE1`? Or alternatively switch to let all tests use the new?